src/services/dexie/settings.ts

```
@@ -245,3 +245,47 @@ export const dbAppSettingsBuildTest = async () => {
+        updatedAt: new Date().toISOString(),
+      };
+    }
+
+    if (section.closing_prayer_auto_assigned.value) {
+      section.closing_prayer_linked_assignment = {
+        value: 'MM_Chairman_A',
+        updatedAt: new Date().toISOString(),
+      };
+    }
+
+    delete section.opening_prayer_auto_assigned;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)